package.json

```diff
@@ -79,6 +79,7 @@
     "gulp-rename": "^1.4.0",
     "gulp-replace": "^1.0.0",
     "gulp-sass": "^4.0.2",
+    "gulp-sass-globbing": "0.0.2",
```

Comment:
The npm page lists four dependencies. Do we not need these? https://www.npmjs.com/package/gulp-sass-globbing

Reply:
The way npm works, if gulp-sass-globbing has dependencies of it's own, it auto-installs those.  If you check the package-lock.json you'll see nested dependency trees.  It prevents you from having to install every dependency of every dependency manually.

Reply:
I'm curious why we're setting all our packages to update to minor releases, why aren't we doing the same for this?
`"gulp-sass-globbing": "^0.0.2",`

Reply:
That's a good idea, I'll add that.  I just used `npm install --save` to add it to the package so that's why it isn't there at the moment.